marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/impl/hub-entities/generateSearchOptions.sjs

```diff
@@ -135,7 +135,73 @@ function generateExplorerWithFacetableAndSortableProperties() {
   ];
 }
 
+function verifySortOperatorsForSortableProperties() {
+  const input = [{
+    "info" : {
+      "title": "Book"
+    },
+    "definitions": {
+      "Book": {
+        "elementRangeIndex": ["bookId"],
+        "properties": {
+          "title": {"datatype": "string", "facetable": true, "sortable": true, "collation": "http://marklogic.com/collation/"},
+          "authors": {"datatype": "array", "sortable": true, "items": {"datatype": "string"}},
+          "bookId": {"datatype": "string", "collation": "http://marklogic.com/collation/"},
+          "completedDate": {"datatype": "dateTime", "facetable": true},
+        }
+      }
+    }
+  }];
+
+  const expOptions = hent.dumpSearchOptions(input, true);
+  return [
+    test.assertEqual("sort", xs.string(fn.head(expOptions.xpath("/*:operator/@name")))),
+    test.assertExists(expOptions.xpath("/*:operator/*:state[@name = 'titleDescending']")),
+    test.assertEqual("descending", xs.string(fn.head(expOptions.xpath("/*:operator[@name = 'sort']/*:state[@name = 'titleDescending']/*:sort-order/@direction")))),
+    test.assertEqual("//*:instance/Book/title", xs.string(fn.head(expOptions.xpath("/*:operator[@name = 'sort']/*:state[@name = 'titleDescending']/*:sort-order/*:path-index")))),
```

Comment:
This is fine to keep here - 5626 will turns this into "/*:envelope/*:instance/*:Book/*:title", just an FYI.